golang-github-osbuild-composer.spec

```diff
@@ -73,11 +75,17 @@ export GOFLAGS=-mod=vendor
 
 TEST_LDFLAGS="${LDFLAGS:-} -B 0x$(od -N 20 -An -tx1 -w100 /dev/urandom | tr -d ' ')"
 
-go test -c -tags=integration -ldflags="${TEST_LDFLAGS}" -o _bin/osbuild-tests %{goipath}/cmd/osbuild-tests
-go test -c -tags=integration -ldflags="${TEST_LDFLAGS}" -o _bin/osbuild-dnf-json-tests %{goipath}/cmd/osbuild-dnf-json-tests
-go test -c -tags=integration -ldflags="${TEST_LDFLAGS}" -o _bin/osbuild-weldr-tests %{goipath}/internal/weldrcheck/
-go test -c -tags=integration -ldflags="${TEST_LDFLAGS}" -o _bin/osbuild-rcm-tests %{goipath}/cmd/osbuild-rcm-tests
-go test -c -tags=integration -ldflags="${TEST_LDFLAGS}" -o _bin/osbuild-image-tests %{goipath}/cmd/osbuild-image-tests
+%define buildtest(o:) %{expand:
+  GO111MODULE=off \\
+  GOPATH=%{gobuilddir}:%{gopath} \\
+  go test -c -tags=integration -ldflags="${TEST_LDFLAGS}" %{-o} %{?*}
```

Comment:
As idea: Should this use the `GOFLAGS` variable defined on line 66?

Reply:
Yeah, but we probably need to change more to make this work on RHEL.